scripts/bash/s3a_wrapper.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a wrapper script for Enceladus run scripts.
+# It is used to resolve the paths to the JCEKS files and
+# to clean up the versions in Dell ECS after the run.
+# To use it just prepend your Enceladus run script by this script
+# Example:
+# ./s3a_wrapper.sh run_standardization_conformance.sh \
+#     --dataset-name "MyDataset"
+#     --dataset-version 1
+#     --report-date "2018-01-01"
+#     --report-version 1
+#
+# If the dataset is published to S3A, the script will find the JCEKS file on its own.
+# If the dataset is not published to S3A, the script will just pass the arguments to the enceladus run script.
+# If the dataset is published to S3A and you pass --jceks-path argument, the script will use the provided JCEKS conf.
+# Format of the --jceks-path argument is:
+#   "spark.hadoop.fs.s3a.bucket.$BUCKET_NAME.security.credential.provider.path=jceks:$JCEKS_PATH"
+#
+# Requirements:
+#   - jq
+#   - curl
+#   - klist
+
+set -e
+
+# Run klist to check for a current Kerberos ticket
+if klist -s; then
+    echo "Kerberos ticket found."
+else
+    echo "No Kerberos ticket found or ticket is expired. Please run kinit."
```

Comment:
```suggestion
    error_exit "No Kerberos ticket found or ticket is expired. Please run kinit."
```

Shouldn't the script exit in this case? Does it make sense to go on?

Reply:
100%. Thank you.